app/routes/lending/components/LendingRequest.tsx

```diff
@@ -0,0 +1,101 @@
+import { usePreparedEffect } from '@webkom/react-prepare';
+import { Helmet } from 'react-helmet-async';
+import { Link, useParams } from 'react-router-dom';
+import { fetchLendingRequest } from 'app/actions/LendingRequestActions';
+import {
+  Content,
+  ContentMain,
+  ContentSection,
+  ContentSidebar,
+} from 'app/components/Content';
+import InfoList from 'app/components/InfoList';
+import NavigationTab, { NavigationLink } from 'app/components/NavigationTab';
+import { FromToTime } from 'app/components/Time';
+import { selectLendingRequestById } from 'app/reducers/lendingRequests';
+import { useAppDispatch, useAppSelector } from 'app/store/hooks';
+
+type Params = {
+  lendingRequestId: string;
+};
+
+const LendingRequest = () => {
+  const { lendingRequestId } = useParams<Params>();
+  const dispatch = useAppDispatch();
+
+  usePreparedEffect(
+    'fetchRequest',
+    () => dispatch(fetchLendingRequest(Number(lendingRequestId))),
```

Comment:
```suggestion
    () => dispatch(fetchLendingRequest(lendingRequestId)),
```

Reply:
Not making it a number gave me a ts-error as the id is from params which is either string or number